Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV BUILD_NUMBER ${BUILD_NUMBER:-1_0_0}
 COPY . .
 RUN ./gradlew clean assemble -Dorg.gradle.daemon=false
 
-FROM eclipse-temurin:19-jre-alpine
```

Comment:
Is it worth building with corretto too? (line 1)

Reply:
Will see if it works locally and if so, do that